<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveAudio</class>
 <widget class="QWidget" name="LiveAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LiveAudio</string>
  </property>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_recording_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start recording</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_recording_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop recording</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="spectrogram">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>20</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="soundcard_idx">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Soundcard</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
